samples/bootloader/README.rst

```diff
@@ -86,37 +70,39 @@ The sample accomplishes a simple and reliable *Root of Trust* (RoT) for a secure
 Provisioning
 ============
 
-The public key hashes are not compiled with the source code of the bootloader sample.
-Instead, they must be stored in a separate memory region through a process called *provisioning*.
+The public key hashes are not compiled with the source code of the NSIB.
+Instead, they must be written to the device in a process called *provisioning*.
+
+The hashes are automatically generated by the build system based on the specified private key and the additional public keys.
 
-By default, the bootloader sample automatically generates and provisions public key hashes directly into the bootloader HEX file, based on the specified private key and the additional public keys.
+By default, the hashes are placed directly into the NSIB HEX file and then automatically provisioned when the HEX file is programmed to the device.
 
-Alternatively, to facilitate the manufacturing process of a device using the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
-If you choose to do so, use the Python scripts located in the :file:`scripts/bootloader` folder to create and provision the keys manually.
+Alternatively, in a more realistic manufacturing process, the NSIB HEX file and the HEX file containing the hashes can be programmed separately.
+For this approach, use the Python scripts located in the :file:`scripts/bootloader` folder.
 
-In both cases, the bootloader accesses the provisioned data using the :ref:`doc_bl_storage` library.
+In either case, the NSIB accesses the provisioned data at runtime using the :ref:`doc_bl_storage` library.
```

Comment:
```suggestion
In either case, the NSIB accesses the provisioned data at run time using the :ref:`doc_bl_storage` library.
```

Reply:
IMHO runtime is more established than "run time". https://en.wikipedia.org/wiki/Runtime_(program_lifecycle_phase)

Reply:
There is a difference in the way we use runtime and run time, according to our style guide: https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-R

run time (n. ) Spelling: Two words. (source: Foldoc)

runtime (mod.) Example: The runtime library is required to build a complete application.

This to me looks more akin to the two-words spelling use case.

Do you think this distinction is incorrect?

Reply:
I was not aware this term was covered by the guide.

I don't think this distinction is correct.

They can be used interchangeably AFAIK, but I don't think the reader will be very confused by "run time", so this suggestion is fine.

Reply:
Ok, so I guess we can continue this discussion offline (if the style guide needs an update)